scanner/families/malware/clam/clam.go

```diff
@@ -201,12 +208,19 @@ func prepareClamDaemon(ctx context.Context, config config.Config) error {
 	logger.Debugf("found clamd binary at: %s", clamDaemonPath)
 
 	// Define default clamd args to run
-	var args []string
+	args := []string{
+		"--config-file", config.ClamDaemonConfigPath,
+	}
```

Comment:
wont work because we create and set config path below and then append it to args again. either reverting to previous changes or moving the whole default clamd configuration code above this works

```suggestion
	// clamd config created here
	
	args := []string{
		"--config-file", config.ClamDaemonConfigPath,
	}
```

Reply:
we can revert to previous version actually, looks better
```suggestion
	var args [] string
```